README.md

```diff
@@ -141,6 +141,42 @@ by a string formatted as:
 
 `<protocol>://[<username>:<password>@]<fqdn>:<port>`
 
+### Timer jobs
+UA client sets up a systemd timer to run jobs that need to be executed recurrently.
+The timer itself ticks every 5 minutes on average, and decides which jobs need
+to be executed based on their _intervals_.
+
+Jobs are executed by the timer script if:
+- They did not run yet, or
+- Their interval is already exceeded.
+
+There is a random delay applied to the timer, to desynchronize job execution time
+on machines spinned at the same time, avoiding multiple synchronized calls to the
+same service.
+
+Current jobs being checked and executed are:
+
+| Job | Description | Interval |
+| --- | ----------- | -------- |
+| update_messaging | Update MOTD and APT messages | 6 hours |
+| update_status | Update UA status | 12 hours |
+| gcp_auto_attach | Try to auto-attach on a GCP instance | 30 minutes |
+
+- The `update_messaging` job makes sure that the MOTD and APT messages match the
+available/enabled services on the system, showing information about available
+packages or security updates. See [MODT messages](#motd-messages).
+- The `update_status` job makes sure the `ua status` command will have the latest
+information even when executed by a non-root user, updating the
+`/var/lib/ubuntu-advantage/status.json` file.
+- The `gcp_auto_attach` job will try to `ua auto-attach` on GCP generic
+Ubuntu machines. If the machine (through the Google Cloud) was converted to
+PRO, it will pick the license and perform the attach. Noops if not on GCP, or
+if already attached.
+
+The timers can be changed in the config file, under `ua_config`, by setting a
```

Comment:
Let's avoid documenting the specific settings inside uaclient.conf since we recommend people use `ua config set`.